docs/contributing/docs-contributions.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Docs Contributions
+---
+
+Gatsby, unsurprisingly, uses Gatsby for its documentation website. Thank you in advance and cheers for contributing to Gatsby documentation! It's people like you that make this community great!
```

Comment:
It might be useful to highlight the number of people who have already done this, to reassure folks that this is a pretty common thing to do.

`git log --pretty="format:%ae" docs/` yields 870 unique emails, so we could say, eg "As of February 2019, over 800 people have contributed to the Gatsby documentation" or whatever makes sense.